datalad_deprecated/resources/website/assets/js/main.js

```diff
@@ -559,9 +565,19 @@ function directory(jQuery, md5) {
         else if (traverse.type === 'search')
           window.location.search = traverse.next;
       });
+      // add visit folder button
+      var crumbs = bread2crumbs(jQuery, md5)
+      var curdir = jQuery(jQuery.parseHTML(crumbs[crumbs.length-1])).attr("href");
+      if (curdir.indexOf("/?dir=")>-1){
+        curdir = (curdir.slice(0,curdir.indexOf("/?dir=")) + curdir.slice(curdir.indexOf("/?dir=")+6));
+      }
+      jQuery('#directory_filter').prepend('<a id="folder-link" href="'+curdir+'">'+
+                                          '<button id="folder-button">Visit Folder</button>'+
```

Comment:
why bother with `<button>` here?  looks quite odd if http://datasets-dev.datalad.org/ has this version:

![image](https://user-images.githubusercontent.com/39889/125854876-ba860790-1e26-45be-8959-4edb6d4a1d7a.png)

also iirc wrapping `button` into `<a>` is also frown upon, or at least that is what I was told [here](https://github.com/sparkletown/sparkle/pull/1387#discussion_r632284901).  So why not just to make it a pure text, like
```suggestion
                                          '[Visit folder]'+
```
?

Reply:
Ah that's right, I'd forgotten about that accessibility caveat -- I initially put it as a button to make it stand out from the breadcrumbs a bit more, so that one would not mistake it as part of the breadcrumbs, but I think I'll just change the styling to accomplish that.

Reply:
As for formatting, that button does look a bit odd, I think it might be the default styling on the browser you are using, I probably should've tested on more browsers. But with just an anchor tag that shouldn't be an issue anymore.